.circleci/config.yml

```diff
@@ -165,73 +153,74 @@ workflows:
           context:
             - hmpps-common-vars
             - hmpps-manage-prison-visits-orchestration-prod
+          filters:
+            branches:
+              only:
+                - main
           requires:
             - request-prod-approval
+      - request-prod-approval:
+          type: approval
+          requires:
+            - deploy_preprod
 
-  build-test-deploy-develop-branch:
+  build-test-deploy-feature:
     jobs:
       - validate:
           filters:
             tags:
               ignore: /.*/
+            branches:
+              ignore:
+                - main
+
```

Comment:
should we look for a branch name with feature in it?

Reply:
comment from @adaviesMOJ on slack - it doesn't need the feature- naming convention, as this flow can now be reused for security patches etc. So we will have 2 flows "main" and "other"